node/coinstacks/gnosis/pulumi/index.ts

```diff
@@ -0,0 +1,95 @@
+import { readFileSync } from 'fs'
+import { deployCoinstack } from '../../../../pulumi/src/coinstack'
+import { Outputs, CoinServiceArgs, getConfig } from '../../../../pulumi/src'
+
+//https://www.pulumi.com/docs/intro/languages/javascript/#entrypoint
+export = async (): Promise<Outputs> => {
+  const appName = 'unchained'
+  const coinstack = 'gnosis'
+  const sampleEnv = readFileSync('../sample.env')
+  const { kubeconfig, config, namespace } = await getConfig()
+
+  const coinServiceArgs = config.statefulService?.services?.map((service): CoinServiceArgs => {
+    switch (service.name) {
+      case 'daemon':
+        return {
+          ...service,
+          ports: {
+            'daemon-http': { port: 8545 },
+            'daemon-ws': { port: 8546 },
+            'daemon-beacon': { port: 8551, ingressRoute: false },
+          },
+          configMapData: {
+            'jwt.hex': readFileSync('../daemon/jwt.hex').toString(),
+          },
+          readinessEndpoint: '/health',
+          volumeMounts: [{ name: 'config-map', mountPath: '/jwt.hex', subPath: 'jwt.hex' }],
+        }
+      case 'daemon-beacon':
+        return {
+          ...service,
+          command: [
+            '/usr/local/bin/lighthouse',
+            'beacon_node',
+            '--network=gnosis',
+            '--disable-upnp',
+            '--datadir=/data',
+            '--port=9000',
```

Comment:
not as concerned about explicit default port for the beacon chain if we just want to use the defaults and save some lines

Reply:
All of these are straight up taken from Gnosis docs, sure we can prune them